pom.xml

```diff
@@ -8,44 +8,215 @@
 		<artifactId>parent-pom</artifactId>
 		<version>1.3.0</version>
 	</parent>
-	<groupId>life.qbic</groupId>
-	<artifactId>postman-core-lib</artifactId>
-	<version>1.0.0-SNAPSHOT</version>
+    <artifactId>postman-core-lib</artifactId>
+	<version>1.0.5-SNAPSHOT</version>
```

Comment:
I'll fix this after it has been reviewed. I wanted to ensure that the deployed version to our nexus got updated -> I wanted to test the classes in postman 0.4 .